package.json

```diff
@@ -3,13 +3,14 @@
   "version": "0.0.0",
   "license": "MIT",
   "dependencies": {
-    "@taquito/taquito": "^11.0.1",
-    "@taquito/signer": "^11.0.1",
     "@taquito/rpc": "^11.0.1",
-    "webpack": "^5.47.1",
-    "webpack-cli": "^4.7.1"
+    "@taquito/signer": "^11.0.1",
+    "@taquito/taquito": "^11.0.1",
+    "rxjs": "^6.6.7"
```

Comment:
Why is rxjs needed?

Reply:
Taquito.. Webpack seems to be failing to find rxjs.

I've tried adding an injection, but it still wasn't able to find it nor did changing the NODE_PATH to point to the taquitos node modules, though that may be explored again here.

Reply:
Okay :ok_hand: